src/blockchain_utilities/blockchain_import.cpp

```diff
@@ -294,6 +292,27 @@ int import_from_file(cryptonote::core& core, const std::string& import_file_path
   // at start_height.
   while (! quit)
   {
+    if (use_batch && !(h % db_batch_size))
+    {
+      uint64_t bytes;
+      uint64_t h2 = 0;
+      bool q2 = false;
+      if (h)
```

Comment:
I'm not sure that works when restarting on a partially imported file: h % x may be non zero, and so batch_stop without a batch_start ?